include/z64.h

```diff
@@ -715,12 +715,14 @@ typedef struct ArenaNode {
     /* 0x04 */ u32 size;
     /* 0x08 */ struct ArenaNode* next;
     /* 0x0C */ struct ArenaNode* prev;
+#if OOT_DEBUG
     /* 0x10 */ const char* filename;
     /* 0x14 */ s32 line;
     /* 0x18 */ OSId threadId;
     /* 0x1C */ Arena* arena;
     /* 0x20 */ OSTime time;
     /* 0x28 */ u8 unk_28[0x30-0x28]; // probably padding
+#endif
```

Comment:
worth noting that the n64 versions have all of this debug information in their heap nodes. obviously cant do anything about that right now, but this `#if` will change later

Reply:
I added a TODO